rasa_core/training/interactive.py

```diff
@@ -997,15 +997,23 @@ def _validate_user_text(latest_message: Dict[Text, Any],
     parse_data = latest_message.get("parse_data", {})
     text = _as_md_message(parse_data)
     intent = parse_data.get("intent", {}).get("name")
+    entities = parse_data.get("entities", [])
+    if entities:
+        entity = []
+        for e in entities:
+            entity.append('[' + e['value'] + '](' + e['entity'] + ')')
+    else:
+        entity = "'None'"
 
     if intent is None:
         print("The NLU classification for '{}' returned '{}'"
               "".format(text, intent))
         return False
     else:
         question = questionary.confirm(
-            "Is the NLU classification for '{}' with intent '{}' correct?"
-            "".format(text, intent))
+            "Is the NLU classification for '{}' with intent '{}' "
+            "and entities: {} correct?"
+            "".format(text, intent, entity))
```

Comment:
ahh sorry about the back and forth on this, but this looks weird now:
` Is the NLU classification for 'i want a [greek](cuisine) restaurant' with intent 'inform' and entities: ['[greek](cuisine)'] correct?`

Can you come up with a better phrasing for this? maybe post it here and we can discuss before you change it again. Something like asking are all marked entities correct and the intent classification

Reply:
Do you think it is better to make this a two step question? first if the intent is classified correctly and then the entity? Or is that too many questions for the user?

Reply:
hmmm i'm not sure which is better now tbh, could you post this in slack and ask for input? I feel like the two step thing might be too many times to hit enter